test/WormholeBase.t.sol

```diff
@@ -8,7 +8,7 @@ import {TestConstants} from "test/Constants.sol";
 
 contract Constructor is Test, TestConstants {
   function testForkFuzz_CorrectlySetsAllArgs(address _wormholeRelayer) public {
-    WormholeBase base = new WormholeBase(_wormholeRelayer);
+    WormholeBase base = new WormholeBase(_wormholeRelayer, msg.sender);
```

Comment:
can we fuzz the owner and assert it's set correctly?

can we also make a new test that makes sure only the owner can change the owner, and not a _notOwner address?

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/94#discussion_r1486860753

Reply:
https://github.com/ScopeLift/l2-flexible-voting/commit/5fb57cdad6d3b0bce2cf8f63c6c873ef27f761ae